index.js

```diff
@@ -6,6 +6,38 @@ var VersionChecker = require('ember-cli-version-checker');
 module.exports = {
   name: 'ember-resolver',
 
+  init: function(tree, options) {
+    this._super.init.apply(this, arguments);
+    this.options = this.options || {};
+
+    var config = this.project.config();
+    var resolverConfig = config.emberResolver || {};
+
+    var resolverFeatureFlags = Object.assign({
+      /* Add default feature flags here */
+    }, resolverConfig.features);
+
+    this.options.babel = {
+      loose: true,
+      plugins: [
+        [require('babel-plugin-debug-macros').default, {
+          debugTools: {
+            source: 'ember-debug'
```

Comment:
We should use the JS module unification module here I think, which would be `@ember/debug`.